cpp/src/parquet/arrow/reader.h

```diff
@@ -51,6 +52,43 @@ class ColumnChunkReader;
 class ColumnReader;
 class RowGroupReader;
 
+static constexpr bool DEFAULT_USE_THREADS = false;
+static constexpr bool DEFAULT_READ_DICTIONARY = false;
+
+class PARQUET_EXPORT ArrowReaderProperties {
```

Comment:
Hm, we might want to use a builder API for this, perhaps mark this experimental for now